src/lib/ngsi/ParseData.h

```diff
@@ -253,14 +263,15 @@ typedef struct UpdateContextData
 *
 * UpdateContextResponseData - 
 */
-typedef struct UpdateContextResponseData
+struct UpdateContextResponseData
 {
+  UpdateContextResponseData(): cerP(0), attributeP(0), metadataP(0), domainMetadataP(0) {}
```

Comment:
NULL is 0, it's just a matter of a typecast.

    #define NULL ((void*) 0)

Sometimes char* is used.

The only problem with NULL is that the lines get longer.